sunscreen_bulletproofs_backend/src/lib.rs

```diff
@@ -0,0 +1,77 @@
+use bulletproofs::{
```

Comment:
Idea: rather than have N crates for each backend, have a single ZKP backend crate with each supplied backend being a feature. Expose traits and types needed to implement a backend in this crate so others can make their own backends.

Reply:
Also, this crate is currently nonsense and isn't hooked up to the frontend.